kfmon.c

```diff
@@ -1457,6 +1470,32 @@ static bool
 			if (thumbnails_count == 3U) {
 				is_processed = true;
 			}
+
+			if (thumbnails_count == 0U) {
```

Comment:
I'd probably make the whole thing branch on whether we're actually running on v5 or not to avoid mixing this with the v3/v4 codepath ;).

Reply:
```
root@ereader:~# cat /mnt/onboard/.kobo/version 
T302440111395,4.9.77,5.2.190625,4.9.77,4.9.77,00000000-0000-0000-0000-000000000691
```

(queried via ssh :D )

Reply:
Cool, thanks ;).